src/View/Crime/createCrime.html.twig

```diff
@@ -1,8 +1,12 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    {% include 'Components/_header.html.twig' %}
-    {% include 'Components/_hashtag.html.twig' %}
-    {% include 'Components/_formCrime.html.twig' %}
+    <div class="main">
+        {% include 'Components/_header.html.twig' %}
+        <div>
```

Comment:
no class on this div, is it really useful ?

Reply:
? I use a class called main